contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -215,7 +215,10 @@ def yarnpkg_command(self, args):
     :rtype: :class:`NodeDistribution.Command`
     """
     return self.Command(
-      bin_dir_path=os.path.join(self.yarnpkg_path, 'bin'), executable='yarnpkg', args=args or [])
+      bin_dir_path=(
```

Comment:
I think that this will fail, because `bin_dir_path` is also prepended to the binary name.

Consider splitting the `Command` args into something like `additional_path_entries` and `binary_name` or something.